packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2690,6 +2690,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     ShapeBorder? shape,
     Clip? clipBehavior,
     BoxConstraints? constraints,
+    bool willDisposeAnimationController = true,
```

Comment:
Maybe a bit more readable
```suggestion
    bool shouldDisposeAnimationController = true,
```

Reply:
done